PWGGA/GammaConv/AliCaloPhotonCuts.cxx

```diff
@@ -2777,7 +2777,25 @@ void AliCaloPhotonCuts::MatchTracksToClusters(AliVEvent* event, Double_t weight,
       EsdTrackCuts = AliESDtrackCuts::GetStandardITSTPCTrackCuts2010();
     // else if run2 data use 2015 PbPb cuts
     }else if (runNumber>=209122){
-      EsdTrackCuts = AliESDtrackCuts::GetStandardITSTPCTrackCuts2015PbPb();
+      // EsdTrackCuts = AliESDtrackCuts::GetStandardITSTPCTrackCuts2015PbPb();
+      // hard coded track cuts for the moment, because AliESDtrackCuts::GetStandardITSTPCTrackCuts2015PbPb() gives spams warnings
+      EsdTrackCuts = new AliESDtrackCuts();
```

Comment:
this is a memory leak, isn't it?!

Reply:
I think this cared for in L.2909ff, no?
```
  if(EsdTrackCuts){
    delete EsdTrackCuts;
    EsdTrackCuts=0x0;
  }
```

Reply:
True, but do we really need the new? As far as I can remember, we had it for quite some time unless someone from offline crew changed it to current state, but I'm not 100% sure any more. Could you please check the history of the file?